### DISCUSSION TITLE
如何升级？

### DISCUSSION THREAD

**Discussion by NatsumiXD** [dreamhunter2333/cloudflare_temp_email#426] [General]
如何在已有的版本上进行升级呢？

**Comment by dreamhunter2333** [comment]
> 如何在已有的版本上进行升级呢？

参考部署过程，再配置一次即可，重复的步骤可以跳过

**Comment by dreamhunter2333** [comment]
已增加文档
https://temp-mail-docs.awsl.uk/zh/guide/quick-start.html
